### PR TITLE
For #30050, setup project wizard shouldn't strip script key

### PR DESCRIPTION
**Pull request by jfboismenu-adsk**
Fixes a regression that was introduced by the security work. Script users are not stripped from the shotgun.yml when configuring a project anymore.

### CODE REVIEW

File: python/tank/deploy/tank_commands/setup_project_wizard.py [hunk 1]
```diff
@@ -641,4 +641,4 @@ def execute(self):
             core_localize.do_localize(self._log, 
                                       self._params.get_configuration_location(sys.platform), 
                                       suppress_prompts=True,
-                                      strip_toolkit_credentials=self._is_session_based_authentication_supported())
```
Comment (jfboismenu-adsk) [inline python/tank/deploy/tank_commands/setup_project_wizard.py]: This was the only instance in the whole codebase where we assigned something other that False to that parameter.

Comment (robblau) [inline python/tank/deploy/tank_commands/setup_project_wizard.py]: Should we remove the ability to do that if it is a dead code path?

Comment (jfboismenu-adsk) [inline python/tank/deploy/tank_commands/setup_project_wizard.py]: It's not used at the moment, but I thought that maybe we could add a step to the adminui module in the future that allowed you to make that decision.

Comment (jfboismenu-adsk) [inline python/tank/deploy/tank_commands/setup_project_wizard.py]: Sorry Rob, I had to merge it in to unblock Jack who was testing the Desktop app. We can remove the dead code later if you feel like having the option in the admin ui is not a good idea.

Comment (robblau) [inline python/tank/deploy/tank_commands/setup_project_wizard.py]: No worries (sorry I've been out of touch this week).

I do think we should get rid of it... I don't think we'll be doing that level of configuration in the wizard, but if we do it is easy enough to add back in.  That is better than having it around now and having to support it forever more.

**Comment by jfboismenu-adsk** [comment]
I stripped the dead code out.

**Comment by manneohrstrom** [comment]
Cr looks :cake: :coffee: :strawberry:

**Pull request closed without merging**